src/clarity-angular/popover/dropdown/dropdown.module.ts

```diff
@@ -9,12 +9,14 @@ import { CommonModule } from "@angular/common";
 import { ClrCommonPopoverModule } from "../common/popover.module";
 import { ClrIconModule } from "../../icon/icon.module";
 import { DROPDOWN_DIRECTIVES } from "./index";
+import { ClrConditonalModule } from "../../utils/conditional/conditional.module";
 
 @NgModule({
     imports: [
         CommonModule,
         ClrCommonPopoverModule,
-        ClrIconModule
+        ClrIconModule,
```

Comment:
Why do you need to import the IconModule? You're never using a `<clr-icon>` in one of the templates, right?
But it does make sense to **export** it from this module (without importing it) if consumers need to manually write an icon every time.